lib/datadog/tracing/contrib/http/circuit_breaker.rb

```diff
@@ -29,6 +29,16 @@ def internal_request?(request)
           end
 
           def should_skip_distributed_tracing?(client_config)
+            if Datadog.configuration.appsec.standalone.enabled
+              active_trace = Tracing.active_trace
+
+              # return true if absence of upstream ASM event (_dd.p.appsec:1)
+              # and no local security event (which sets _dd.p.appsec:1 locally)
+              no_upstream_or_local_event = active_trace.nil? ||
+                active_trace.get_tag(Datadog::AppSec::Ext::TAG_APPSEC_EVENT) != '1'
+              return true if no_upstream_or_local_event
```

Comment:
how about this?

```ruby
return true unless Tracing.active_trace

Tracing.active_trace.get_tag(Datadog::AppSec::Ext::TAG_APPSEC_EVENT) != '1'
```

Reply:
I did something similar (that would also satisfy Rubocop) thanks !